verticapy/utilities.py

```diff
@@ -465,7 +472,8 @@ def pandas_to_vertica(
 cursor: DBcursor, optional
     Vertica database cursor.
 name: str, optional
-    Name of the new relation. If empty, a temporary local table is created.
+    Name of the new relation or the relation to insert the data in. 
+    If empty, a temporary local table is created.
 schema: str, optional
     Schema of the new relation. The default schema is public.
```

Comment:
Wouldn't it be better to make the default schema to be set by `temp_schema` defined in __init__.py?

Reply:
Yes it can be modified in the next PR.